### PR TITLE
[zh] sync ephemeral-volumes storage-classes windows-storage

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -421,18 +421,14 @@ same namespace, so that these conflicts can't occur.
...
+if they can create Pods, even if they do not have permission to create PVCs directly.
+Cluster administrators must be aware of this. If this does not fit their security model,
+they should use an [admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/)
 -->
-启用 GenericEphemeralVolume 特性会有一些副作用，用户能创建 Pod 就能间接地创建 PVC，
-即使他们没有权限直接创建 PVC。
-集群管理员必须意识到这一点。
-如果这不符合他们的安全模型，他们应该使用[准入 Webhook](/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers/)
-拒绝包含通用临时卷的对象，例如 Pod。
+只要用户有权限创建 Pod，就可以使用通用的临时卷间接地创建持久卷声明（PVCs），
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/ephemeral-volumes.md:429]:
```suggestion
只要用户有权限创建 Pod，就可以使用通用的临时卷间接地创建持久卷申领（PVCs），
```

File: content/zh-cn/docs/concepts/storage/ephemeral-volumes.md [hunk 2]
```diff
@@ -421,18 +421,14 @@ same namespace, so that these conflicts can't occur.
...
+即使他们没有权限直接创建 PVCs。集群管理员必须注意这一点。如果这不适合他们的安全模型，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/ephemeral-volumes.md:430]:
```suggestion
即使他们没有权限直接创建 PVCs。集群管理员必须注意这一点。如果这与他们的安全模型相悖，
```

File: content/zh-cn/docs/concepts/storage/ephemeral-volumes.md [hunk 3]
```diff
@@ -421,18 +421,14 @@ same namespace, so that these conflicts can't occur.
...
+他们应该使用一个 [admission webhook](/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers/)。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/ephemeral-volumes.md:431]:
```suggestion
他们应该使用[准入 Webhook](/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers/)。
```

**Review by ydFu — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**